src/edu/gatech/cs2340/trydent/ContinuousEvent.java

```diff
@@ -0,0 +1,59 @@
+package edu.gatech.cs2340.trydent;
+
+/**
+ * Represents an event that is called by the engine every frame.
+ * @author Garrett Malmquist
+ *
+ */
+public abstract class ContinuousEvent {
+	
+	private boolean started = false;
+	
+	public ContinuousEvent() {
+		TrydentEngine.addContinuousEvent(this);
+	}
+	
+	/**
+	 * Called by the engine on update.
+	 */
+	protected final void doUpdate() {
```

Comment:
I think this should probably have package scope because it seems to be an engine internal method. Also, this method can't be overridden because it is final.
Notably, whatever scope this has, it should probably be the same as doStop.